src/main/java/com/playmonumenta/worlds/paper/MonumentaWorldManagementAPI.java

```diff
@@ -108,74 +109,65 @@ public static void sortWorld(Player player) throws Exception {
 		player.saveData();
 
 		// Figure out what world the player would sort to
-		World newWorld = listener.getSortWorld(player, player.getWorld().getName());
+		World newWorld = listener.getSortWorld(player);
 
 		// Move the player to that world at their last position (or world spawn)
 		MonumentaRedisSyncAPI.getPlayerWorldData(player, newWorld).applyToPlayer(player);
 	}
 
-	/**
-	 * @deprecated
-	 * This method can no longer be called async (2nd parameter) and will be removed from the API eventually. Use the one with two arguments.
-	 */
-	@Deprecated
-	public static World ensureWorldLoaded(String worldName, boolean unused, boolean copyTemplateIfNotExist) throws Exception {
-		return ensureWorldLoaded(worldName, copyTemplateIfNotExist);
-	}
```

Comment:
Can we add this one back in, and add the original version of the below one too (also deprecated), and have them call the new API with null? that way old stuff will still work with the old APIs.

Reply:
The first one is doable, but the second one is causing IntelliJ to insist that any attempt to call `ensureWorldLoaded(String, null)` is `ensureWorldLoaded(String, boolean)` instead of `ensureWorldLoaded(String, String)`. No clue why, but it looks like "will be removed some day" needs to be "about now", unless we rename the modern method to something else.